companion/src/firmwares/opentx/opentxinterface.cpp

```diff
@@ -344,6 +344,13 @@ bool OpenTxEepromInterface::load(RadioData &radioData, const uint8_t *eeprom, in
     return false;
   }
 
+  // we do not want to support OpenTX 2.1 ans later on Companion 2.0
```

Comment:
I think that this check should be in 
'''
bool OpenTxEepromInterface::checkVersion(unsigned int version)
'''